### PR TITLE
Restore "nominate a teacher" trophy banner

### PR DESCRIPTION
**Pull request by bencodeorg**
Adds "nominate a teacher" banner back to the signed out homepage and the top of the teacher dashboard (using Molly's new announcement controls!)

**Signed out homepage**

image

**Teacher dashboard banner**

image

Main nit right now is that background color for the text section of the banner clashes with the image. A couple options to fix:
1. Make all banners the color that matches the image, like [here](https://github.com/code-dot-org/code-dot-org/pull/27306) -- challenge is that the same banner is used in places like the [teacher courses page](https://studio.code.org/courses?view=teacher), which would introduce a third blue to the page.
2. Make background color configurable as part of Molly's announcement controls.

## Testing story

Tested locally -- used image served from my local server for testing, but set image URL to use production path once it's deployed with this change. Tested using other images in the same directory to confirm that when I navigate directly to the page, I get the image as I'd expect ([example here](https://code.org/shared/images/fill-970x562/teacher-announcement/hoc2019-posthoc.jpg)).

### CODE REVIEW

**Review by islemaster — APPROVED** [review]
:+1: Thanks for picking this up Ben!  It looks great.  I'd love to get this merged and then come back for the color fix so that, worst case, we have something out the door in tomorrow's DTP.

**Comment by bencodeorg** [comment]
Can do, will wait for drone and merge early tomorrow.

**Comment by bencodeorg** [comment]
Also note in case anyone's curious Brendan advised that pegasus will automagically take a .png file and convert it to .jpg (as I'm doing here).

**Review by molly-moen — APPROVED** [review]
Nice!

File: pegasus/sites.v3/code.org/public/index.haml [hunk 1]
```diff
@@ -27,7 +27,7 @@ style_min: true
   - if request.language == "en"
     = view :curriculum_banner
     = view :homepage_below_hero_announcement
-    = view :homepage_below_hero_plane_banner
+    = view :homepage_below_hero_trophy_banner
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/index.haml:30]: Glad I left the partial in the source tree.  Had a feeling we'd need it again :)

Comment (bencodeorg) [reply pegasus/sites.v3/code.org/public/index.haml:30]: Yes, thank you made it very easy!

**Pull request merged**